docs/docs/developer-guide.md

```diff
@@ -74,29 +68,25 @@ After spinning the example up, you may head to `localhost:8888` and use one of t
 
 ## Working with SQLx
 
-This crate uses sqlx. For development and compilation a Postgres Database is required. You can use Docker to launch
-one.:
-
-```sh
-docker run -d --name postgres-16 -p 5432:5432 -e POSTGRES_PASSWORD=postgres postgres:16
-```
-The `crates/lakekeeper` folder contains a `.env.sample` File.
-Copy this file to `.env` and add your database credentials if they differ.
+This crate uses sqlx. For development and compilation a Postgres Database is required. This is part of the [Initial setup](#initial-setup).
+If your database credentials used differ, please modify the `.env` accordingly and run `source .env` again.
 
 Run:
-
 ```sh
-sqlx database create
-sqlx migrate run
-```
+# migrate db. Make sure you have sqlx-install with `cargo install sqlx-cli`
+cd crates/lakekeeper
+sqlx database create && sqlx migrate run
 
-Before committing changes to SQL queries, run: 
+just sqlx-prepare
 
-```sh
-`just sqlx-prepare`
-```
```

Comment:
If you changed any of the SQL statements embedded in Rust code, run this before pushing to GitHub.